src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -196,18 +196,18 @@ public function preprocessEvent(Event $event, string $langcode, array $settings)
         ];
 
         // Search for language keywords. Take the highest value of all items that match..
-        $totalLanguageIcons = 0;
+        $totalLanguageIcons = [];
         if (!empty($variables['labels'])) {
             foreach ($languageIconKeywords as $keyword => $value) {
                 if (in_array($keyword, $variables['labels'])) {
-                    $totalLanguageIcons = $value;
+                    array_push($totalLanguageIcons, $value);
```

Comment:
Nit: You could just do `$totalLanguageIcons[] = $value;`
`array_push` is mostly used when you want to append multiple values with one call.
But there's technically nothing wrong with using it here, just pointing it out.

Reply:
thx for the hint